api/cds.proto

```diff
@@ -164,7 +190,10 @@ message Cluster {
     V4_ONLY = 1;
     V6_ONLY = 2;
   }
-  DnsLookupFamily dns_lookup_family = 17;
+
+  // The DNS IP address resolution policy. If this setting is not specified, the
+  // value defaults to V4_ONLY.
```

Comment:
Link to enum values in the above blocks. I think everywhere in this file this would be useful.